concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
This is not 100% guaranteed and there is small race condition/window (depending on how often _eventually_ checks the function results). Nevertheless, why would we _not_ want to run smoke/acceptance/other tests if the availability tests have failed? I wouldn't change the previous behaviour and still run them (simply after post-deploy, without depending on `avail tests`).